File: MyWallet/Front_MyWallet/src/app/_model/transaction.ts
export class Transaction {
  id: number;
  name: string;
  categoryId: number;
  walletId: number;
  bankId: number;
  amount: number;
  timestamp: string;

  public static of(name: string, categoryId: number, walletId: number, bankId: number, amount: number, timestamp: string) {
    let transaction: Transaction = new Transaction();
    transaction.name = name;
    transaction.categoryId = categoryId;
    transaction.walletId = walletId;
    transaction.bankId = bankId;
    transaction.amount = amount;
    transaction.timestamp = timestamp;
    return transaction;
  }
}

// "2018-07-14T17:45:55.948354"

File: MyWallet/Backend_MyWallet/src/main/java/org/koz/wallet/app/transaction/model/Transaction.java
package org.koz.wallet.app.transaction.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Transaction {
    private Long id;
    private Long userId;
    private String name;
    private Long categoryId;
    private Long walletId;
    private Long bankId;
    private BigDecimal amount;
    private LocalDateTime timestamp;
}